### PR TITLE
[player-395] showing flash alerts when flash is disabled in the browser

### PR DESCRIPTION
**Pull request by ksunika**
[PLAYER-395](https://jira.corp.ooyala.com/browse/PLAYER-395).

### CODE REVIEW

**Comment by aeng7** [comment]
Can we add some unit tests for this feature?

File: js/controller.js [hunk 1]
```diff
@@ -668,6 +697,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         width: ""
       });
       this.state.forceControlBarVisible = (this.state.pluginsElement.children().length > 0);
+
+      if (this.state.midVideo) {
+        this.mb.publish(OO.EVENTS.PAUSE);
```
Comment (aeng7) [inline js/controller.js]: How come we pause the video here (and resume above in adsPlayed)?

Comment (ksunika) [inline js/controller.js]: Until we enable the flash, the call to pause the main content doesn't happen, so we have the flash alert visible and the main video playing in the background unless we pause it.

File: js/controller.js [hunk 2]
```diff
@@ -656,6 +680,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.state.isPlayingAd = false;
       this.state.pluginsElement.removeClass("oo-showing");
       this.state.pluginsClickElement.removeClass("oo-showing");
+      this.showResponsiveDiv();
+      this.changeIMAiframeOpacity(0);
```
Comment (ksunika) [inline js/controller.js]: I think it's because we manually change the opacity to 1 before that, it doesn't change back to 0 by itself.

**Pull request merged**